backend/app/tests/crud/evaluations/test_core.py

```
@@ -0,0 +1,629 @@
+from app.tests.utils.test_data import create_test_evaluation_dataset
+
+
+class TestCreateEvaluationRun:
+    """Test creating evaluation runs."""
+
+    def test_create_evaluation_run_minimal(self, db: Session, test_dataset_factory):
+        """Test creating an evaluation run with minimal config."""
+        eval_run = create_evaluation_run(
+            session=db,
+            run_name="test_run",
+            dataset_name=test_dataset.name,
```

Ruff (0.14.8)

Undefined name `test_dataset`

(F821)